pokemonterminal/main.py

```diff
@@ -110,28 +69,40 @@ def main(argv=None):
         print("Dry run, exiting.")
         return
 
-    if options.clear:
-        if PIPE_EXISTS:
-            pipe_out = os.open(PIPE_PATH, os.O_WRONLY)
-            os.write(pipe_out, b"quit\n")
-            os.close(pipe_out)
-        scripter.clear_terminal()
-        return
-
-    if is_slideshow and options.id <= 0 and size > 1:
-        if os.name == 'nt':
-            print("Slideshow not supported on Windows yet.")
-            sys.exit(0)
-        if PIPE_EXISTS:
-            print("Slideshow already running in this instance!")
-            sys.exit(0)
-        if options.slideshow <= 0:
-            print("Time has to be greater then 0. You can use decimal values.")
+    e = create_named_event("Pokemon-Terminal_Wallpaper" if options.wallpaper else "Pokemon-Terminal_Terminal")
+    try:
+        if options.clear:
+            if e.is_duplicate():
```

Comment:
I don't think `PosixNamedEvent.__is_duplicate` is getting set properly to ever hit this case